src/index.ts

```diff
@@ -116,10 +120,16 @@ export class Tracker {
     }
   }
 
+  private count(key: string): void {
+    if (this.options.debug) {
+      document.body.dataset[key] = String(Number(document.body.dataset[key] || 0) + 1)
+    }
+  }
+
   private flush(): void {
     const queue = this.eventQueue;
-    if (queue.length) {
-      this.log("Flushing events...");
+    if (this.options.debug) {
+      console.group("[@ridi/event-tracker] Flushing events...");
```

Comment:
```suggestion
    if (queue.length) {
      this.log("Flushing events...");
```
원래 while 문 안에서 발생하는 로그들을 그룹 안에 넣으려는 목적은 아니었습니다.
그냥 `Flushing events...` 메세지를 굵은 글씨로 보고 싶어서 그랬던 건데요.
그것보다는, `log` 나 `logEvents` 를 통하지 않고 직접 로그 메세지를 출력하는 것을 피하고 싶습니다.
원래대로 `this.log` 를 호출하는 것이 좋겠습니다.

Reply:
로컬에서 디버깅을 해보니 그룹으로 묶는게 파악하는데 훨씬 수월해서 수정했습니다.
기존 방식은 콘솔 점유율(?)이 너무 높아서 개발에 방해가 되는 경우가 많아 보통 MainTrackerOptions를 debug: false 로 두고 개발하는 경우가 많았습니다.

Reply:
알겠습니다. 요건 따로 리팩토링이 필요하겠네요.